README.rst

```diff
@@ -0,0 +1,86 @@
+xss-utils
+=============================
+
+|pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Utilities to prevent possible XSS attacks on Django/Mako templates.
```

Comment:
This will ultimately have things to protect Python, JavaScript and other templating engines.  Could be written more generically with some examples.  Like `...attacks in the Open edX codebase (e.g. in Python, JavaScript, Django templates, Mako templates, etc.).